server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/StartAuditRound.java

```diff
@@ -314,7 +314,12 @@ public String startRound(final Request the_request, final Response the_response)
     final DoSDashboard dosdb = Persistence.getByID(DoSDashboard.ID, DoSDashboard.class);
     final BigDecimal riskLimit = dosdb.auditInfo().riskLimit();
     final String seed = dosdb.auditInfo().seed();
-    final List<ComparisonAudit> comparisonAudits = Persistence.getAll(ComparisonAudit.class);
+
+    final List<ComparisonAudit> comparisonAudits =
+      Persistence.getAll(ComparisonAudit.class).stream()
+      .filter(ca -> ca.isTargeted() && !ca.isFinished())
```

Comment:
We'll be trying to make samples for fewer contests - `OPPORTUNISTIC_BENEFITS` are now filtered by way of the `isTargeted()`.